.github/workflows/build.yml

```diff
@@ -127,7 +127,7 @@ jobs:
             # markdown
             libmarkdown2-dev
             # markdown/webhelper
-            libwebkitgtk-3.0-dev
+            libwebkit2gtk-4.0-dev
```

Comment:
IIUC Webkit2 is not a drop in replacement for Webkit, their APIs diverge, so unless those plugins are updated to webkit2 this is wrong.

Reply:
`libwebkit2gtk-4.0-dev` is the dependency being used in the current Ubuntu [`geany-plugins`](http://archive.ubuntu.com/ubuntu/pool/universe/g/geany-plugins/geany-plugins_1.38+dfsg-1build1.debian.tar.xz) package.  `libwebkitgtk-3.0-dev` doesn't exist anymore after 18.04.

Reply:
I was aware that webkit was no longer available, but IIUC the plugins have not yet been updated (see #1217 and #1224)

Reply:
The Markdown plugin does work with `libwebkit2gtk-4.0-dev`.   GeanyPy is gone for other reasons.  I don't know about MultiTerm, I vaguely recall it had other issues.  WebHelper, I understood was using an even older version of webkitgtk than Markdown.

Reply:
Ok, since markdown is already updated (but didn't update this) and webhelper is updated by #1224 (and it doesn't update this either) I guess its ok to make this change.  Certainly leaving webkit1 there is useless since its not in the distro.

P.S. I confused webhelper here and devhelper in #1217, ignore references to it other than multiterm